src/gmt_io.c

```diff
@@ -7113,6 +7115,10 @@ int gmt_parse_segment_header (struct GMT_CTRL *GMT, char *header, struct GMT_PAL
 
 	if (!header || !header[0]) return (0);
 
+	if (gmt_parse_segment_item (GMT, header, "-t", line)) {	/* Found a potential -t option for transparency */
+		transparency = atof (line) * 0.01;
+		got_transparency = true;	/* Must apply to any fill and pen settings */
+	}
```

Comment:
Does this mean **plot** and **plot3d** can understand **-t** in the segment header? If yes, you may want to add **-t** to https://docs.generic-mapping-tools.org/dev/plot.html#segment-header-parsing.

Reply:
Yes, I was debating if it should be a hidden trick for psevents to work or an expansion of capability for all.  I guess it is a reasonable extension so I will update the docs.

Reply:
OK, done